### DISCUSSION TITLE
Try pybricks before purchasing

### DISCUSSION THREAD

**Discussion by dss0427** [pybricks/support#1812] [General]
I'm interested in pybricks for the FLL team that I'm coaching.  I'd like to try something with pybricks before purchasing.  In the documentation for running your first program it says to click the plus sign for a new project.  There is no plus sign in my window.  On the pricing page there is a suggestion to try before buying a license.  How do I do that?

**Comment by BertLindeman** [comment]
Hello,

About the plus, I am not sure if you mean this, but in code.pybricks.com on the left side there are two icons. 
- the top one is "files"
- the second is "configuration"

After I pressed the file icon I see:
![afbeelding](https://github.com/user-attachments/assets/a73d6b19-62df-4c0b-8360-a184a7a219c0)

And I forgot to indicate the plus, but you can see on the left side of the bluetooth button the "plus".

Bert

**Reply by dss0427** [reply]
Thanks.  I did find the add program (plus sign), but was unable to do anything without signing up in patreon or by buying a license.  I ended up buying a license, and am now able to create programs.  Is there a good source of examples.  There seems to be a limited number of youtube videos.

**Comment by BertLindeman** [comment]
Hello,

I assume you already found [pybricks getting started](https://pybricks.com/learn/) 
It is still work in progress, but on the top of the page there is a button "projects".
That page has various projects. One of them is [projects using block coding](https://pybricks.com/project/block-coding/) 

Welcome aboard,

Bert
